modules/core/quick/farm_quick.install

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Update functions for the farmOS Quick Form module.
+ */
+
+declare(strict_types=1);
+
+/**
+ * Install the new quick_form entity type.
+ */
+function farm_quick_update_9001(): void {
```

Comment:
Can we change this to an implementation of `hook_post_update_NAME()` instead of `hook_update_N()`? We've adopted the former as our default update hook and have not used `hook_update_N()` in 2.x at all (unless I'm forgetting one).

This also means it needs to move from `farm_quick.install` to `farm_quick.post_update.php`.

Reply:
Moved the implementation to a post_update hook.